packages/web/src/pages/[propertyAddress].tsx

```diff
@@ -12,6 +13,7 @@ const PropertyAddressDetail = (_: Props) => {
   return (
     <div>
       <PropertyHeader propertyAddress={propertyAddress} />
+      <AssetOutline metricsAddress={propertyAddress} marketAddress={propertyAddress} />
```

Comment:
@abyssparanoia 
I didn't understand what "metrics address" and "market address" meant when I read the requirements document. Do you know what these are?